viewer/src/main/webapp/viewer-html/components/AttributeList.js

```diff
@@ -363,12 +363,14 @@ Ext.define ("viewer.components.AttributeList",{
             this.download();
         }
     },
-    filterChanged : function (filter,appLayer){
-        if (this.layerSelector !== null){
-            var selectedLayer = this.layerSelector.getValue();
-            if(selectedLayer){
-                if(selectedLayer.id == appLayer.id){
-                   this.loadAttributes(appLayer);
+    filterChanged : function (filter,appLayer) {
+        if (!Ext.Object.isEmpty(this.grids)) {
```

Comment:
To prevent deep nesting of if statements, it would be better to write something like:
```
if (!Ext.Object.isEmpty(this.grids)) {
    return;
}
```
instead of surrounding the existing block in another if statement